code/modules/power/power.dm

```diff
@@ -289,14 +289,9 @@
 			SEND_SIGNAL(src, COMSIG_MACHINERY_POWER_LOST)
 			. = TRUE
 
-	if(appearance_power_state != (machine_stat & NOPOWER))
+	if(!(machine_stat & NOPOWER))
 		update_appearance()
 
-// Saves like 300ms of init by not duping calls in the above proc
-/obj/machinery/update_appearance(updates)
-	. = ..()
-	appearance_power_state = machine_stat & NOPOWER
-
```

Comment:
This code was an optimization of appearance updates. update_appearance creates a change when called in the above proc only when the power level is different, so this code avoids redundant calls. This is no longer the same thing, instead of looking for a change you are now looking for a powered machine

Reply:
Yup i'm adding it back now